eng/common/pipelines/templates/steps/docs-metadata-release.yml

```diff
@@ -48,7 +48,7 @@ steps:
       -WorkDirectory "${{ parameters.WorkingDirectory }}"
       -DocRepoLocation "${{ parameters.WorkingDirectory }}/repo"
       -Language "${{parameters.Language}}"
-      -DocRepoContentLocation ${{ parameters.DocRepoDestinationPath }}
+      -Configs "${{ parameters.CIConfigs }}"
```

Comment:
What does the Config pass here?

Reply:
Answered offline. For anyone else looking, [this PR](https://github.com/Azure/azure-sdk-for-java/pull/17938) answers the question.